toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -83,6 +84,38 @@ else
     retry "{ mount -L $LABEL -o ro $ISO_ROOT || grep -qs $ISO_ROOT /proc/mounts ; }"
 fi
 
+# If we are running unattended (i.e., have $UNATTENDED_CONFIG_FILE), we need to make sure that we only run once.
+# We can use flock to synchronize the unattended install flows.
+if [[ -f "$UNATTENDED_CONFIG_FILE" ]]; then
```

Comment:
Could we combine this `if` with the one behind it with an `else`? That way it should be a bit clearer, that there's nothing special happening in this `if` (like removal of the unattended config file), that would influence the result of the second check.

Reply:
I considered it, but I feel these are really unrelated bits of code. I found it much more readable to keep them split into their own blocks.